test/e2e/pages/vl-template.page.js

```diff
@@ -0,0 +1,16 @@
+const VlTemplate = require('../components/vl-template');
+const { Page } = require('vl-ui-core');
+const { Config } = require('vl-ui-core');
+
+class VlTemplatePage extends Page {
+    async getTemplate() {
+        return new VlTemplate(this.driver, 'vl-template');
+    }
+
+    async load() {
+        //await super.load(Config.baseUrl + '/demo/vl-template.html');
```

Comment:
nog nodig?

Reply:
dat is een reminder omdat we wachten op webuniversum zodat het terug op poort 8080 kan draaien. Dit is een quick fix zodat ik geen aanpassingen moet doen aan de core en andere componenten impacteer. Besproken met @coemans .